master/internal/trial.go

```diff
@@ -787,19 +784,6 @@ func (t *trial) allReady(ctx *actor.Context) bool {
 		return false
```

Comment:
I have a proposed change on the comment of this function:

```
// allReady returns true if and only if an appropriate ContainerStarted message and a corresponding
// containerConnected message have been received from each container in the trial. The two messages
// are not guaranteed to come in-order.
func (t *trial) allReady(ctx *actor.Context) bool {
     // If a trial has passed allReady it can never return to a state of not ready until the
     // current containers are all terminated.
```
changed to be
```
// allReady returns true if and only if all the containers are reported to be started with the 
// ContainerStarted message and their sockets to be connected with the containerConnected 
// message. The two messages are not guaranteed to come in-order. During each run of the 
// trial, once all the containers are ready this function will return true afterward because this 
// function is used in deciding if needs killing when terminating the trial.
func (t *trial) allReady(ctx *actor.Context) bool {
```

Reply:
changed it